.travis.yml

```diff
@@ -1,5 +1,7 @@
 # https://travis-ci.org
 
+sudo: required
```

Comment:
travis do some changes the some year, and the proyect before of specific year (i don't remember it specifictly, 2 or 1), and the new proyects added in that time need include this line to works if it requieres root, in the proyect don't affects this but forks yes, and is very problematic when at least i try to make a pull request, any pr fails but if i do a change to can works with travis that commit will be reflacted in the pr, finally i think its better inlcude it and simplify this.